### PR TITLE
feat(organizations-access): new command to manage access for individu…

### PR DESCRIPTION
**Pull request by boennemann**
…al orgs

@janl @christophwitzko

### CODE REVIEW

File: src/commands.js [hunk 1]
```diff
@@ -36,7 +37,9 @@ exports.aliases = {
   signin: 'login',
   signout: 'logout',
   on: 'enable',
-  off: 'disable'
+  off: 'disable',
+  access: 'organization-access',
+  'organisation-access': 'organization-access'
```
Comment (janl) [inline src/commands.js]: + `org-access`

File: src/organization-access.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = function (flags) {
+  log.verbose('organization-access', 'starting command')
+
+  if (!flags.token) {
+    log.info('organization-access', 'Not logged in.')
+    process.exit(1)
+  }
+
+  log.info('organization-access', 'Opening GitHub application settings in browser')
```
Comment (janl) [inline src/organization-access.js]: maybe export strings to `src/lib/story.js`?

**Comment by boennemann** [comment]
@janl @christophwitzko pls review again/merge :)

**Comment by christophwitzko** [comment]
LGTM :+1:

**Comment by boennemann** [comment]
With @janl's issues addressed and @christophwitzko's +1 I'm going to release this now.

**Pull request merged**